src/chrome/content/rules/Cucumber.io.xml

```diff
@@ -0,0 +1,12 @@
+<ruleset name="Cucumber.io">
+
+	<target host="cucumber.io" />
+	<target host="clinic.cucumber.io" />
+	<target host="cukenfest.cucumber.io" />
+	<target host="docs.cucumber.io" />
+	<target host="handbook.cucumber.io" />
+	<target host="thoughts.cucumber.io" />
```

Comment:
```
aruba.cucumber.io
shouty.cucumber.io
```

Reply:
For future reference, `aruba.cucumber.io` has active mixed content from `fonts.googleapis.com`